presto-main/src/test/java/com/facebook/presto/execution/TestTaskExecutor.java

```diff
@@ -115,7 +115,7 @@ public void test()
         }
     }
 
-    @Test
+    @Test(invocationCount = 10000)
```

Comment:
The race is due to the test being written incorrectly though, not a race in the Presto code. I would remove it, since our tests are already taking too long :)